tools/floodgate/js/floodgate.js

```diff
@@ -10,16 +11,44 @@ import {
 import {
   updateProjectInfo,
   updateProjectDetailsUI,
+  updateProjectStatusUIFromAction,
   updateProjectStatusUI,
 } from './ui.js';
-import promoteFloodgatedFiles from './promote.js';
-import floodgateContent from './copy.js';
 
 async function reloadProject() {
   loadingON('Purging project file cache and reloading... please wait');
   await purgeAndReloadProjectFile();
 }
 
+async function floodgateContentAction(project, config) {
+  const params = getParams(project, config);
+  params.spToken = getAccessToken();
+  const copyStatus = await postData(config.sp.aioCopyAction, params);
```

Comment:
@sirivuri The app based auth for the FG actions got finalized this morning. This will change the API signature and auth mechanism. The SP token will no longer required to be passed in as part of the payload when calling AIO actions. Let's touch base offline to discuss the specifics.